tests/integration/docusaurus/validation/checkpoints/how_to_pass_an_in_memory_dataframe_to_a_checkpoint.py

```diff
@@ -15,10 +14,8 @@
     df, asset_name="taxi_frame", batch_metadata={"year": "2019", "month": "01"}
 )
 
-checkpoint = SimpleCheckpoint(
-    name="my_taxi_validator_checkpoint",
-    data_context=context,
-    validator=validator,
+checkpoint = context.add_or_update_checkpoint(
+    name="my_taxi_validator_checkpoint", validator=validator
 )
 
 checkpoint_result = checkpoint.run()
```

Comment:
why does this fail now?

Reply:
CLI to run: `pytest --docs-tests  -k "checkpoint" tests/integration/test_script_runner.py`

Reply:
Fixed for now with the call to save_expectation_suite

Reply:
💙